mappings/net/minecraft/entity/mob/warden/WardenEntity.mapping

```diff
@@ -0,0 +1,70 @@
+CLASS net/minecraft/unmapped/C_fynaplbn net/minecraft/entity/mob/warden/WardenEntity
+	FIELD f_aklkpyqr lastHeartAnimationTicks I
+	FIELD f_azgepkis sniffingAnimation Lnet/minecraft/unmapped/C_kxntavoz;
+	FIELD f_bglhpwqx DIGGING_PARTICLE_OFFSET F
+	FIELD f_bokpqluc DARKNESS_DISPLAY_LIMIT I
+	FIELD f_catvnevx VIRBRATION_COOLDOWN_TICKS I
+	FIELD f_ceomecbq ANGER_BOOST_ON_HURT I
+	FIELD f_ctsqwubn emergingAnimation Lnet/minecraft/unmapped/C_kxntavoz;
+	FIELD f_ejgyiohn ATTACK_KNOCKBACK F
+	FIELD f_fmvpjdup PROJECTILE_ANGER I
+	FIELD f_gmstpsno ANGER_TICK_DELAY I
+	FIELD f_gqdmtuqw MAX_HEALTH I
+	FIELD f_jaxcsvmw gameEventListener Lnet/minecraft/unmapped/C_wpaexsvv;
+	FIELD f_jctlypxu roaringAnimation Lnet/minecraft/unmapped/C_kxntavoz;
+	FIELD f_jgqxnrro DEFAULT_ANGER_LEVEL I
+	FIELD f_jhswzlgv tendrilAnimationTicks I
+	FIELD f_knxwlbyj DARKNESS_INTERVAL I
+	FIELD f_kvtjaaga DIGGLE_PARTICLE_DURATION F
+	FIELD f_lgkcpptz ATTACK_DAMAGE I
+	FIELD f_lvckwbmg KNOCKBACK_RESISTANCE F
+	FIELD f_lyqrfgah DARKNESS_DURATION I
+	FIELD f_ngwmfivw EVENT_LISTENER_RANGE I
+	FIELD f_nmetugak diggingAnimation Lnet/minecraft/unmapped/C_kxntavoz;
+	FIELD f_nukmqgnz lastTendrilAnimationTicks I
+	FIELD f_pbinxyrw attackingAnimation Lnet/minecraft/unmapped/C_kxntavoz;
+	FIELD f_poriirpd PROJECTILE_ANGER_DISTANCE I
+	FIELD f_qjoaogsr TOUCH_COOLDOWN_TICKS I
+	FIELD f_rtydirkh sonicBoomAnimation Lnet/minecraft/unmapped/C_kxntavoz;
+	FIELD f_upngnwky DARKNESS_RADIUS I
+	FIELD f_vwjcyten ANGER_LEVEL Lnet/minecraft/unmapped/C_rinmcaxy;
+	FIELD f_wogyfxys PROJECTILE_TICK_THRESHOLD I
+	FIELD f_wpbqvehb LOGGER Lorg/slf4j/Logger;
+	FIELD f_wpywmvqz FIGHTING_MOVEMENT_SPEED F
+	FIELD f_xhbuduik DIGGING_PARTICLE_COUNT I
+	FIELD f_xkfszxjq heartAnimationTicks I
+	FIELD f_zjwrueat MELEE_UNTIL_SONIC_BOOM_TICKS I
```

Comment:
Can you clarify this?
Seems like it should be `TICKS_UNTIL_SONIC_BOOM`, but also melee is there too?

Reply:
It's the time the warden can use melee attacks until it can sonic boom again i believe. Mojmap is `TIME_TO_USE_MELEE_UNTIL_SONIC_BOOM` but that's unnecessarily verbose imo so I tried to shorten it